velox/functions/lib/aggregates/SingleValueAccumulator.h

```diff
@@ -33,6 +33,16 @@ struct SingleValueAccumulator {
 
   bool hasValue() const;
 
+  /// Returns 0 if stored and new values are equal; <0 if stored value is less
+  /// then new value; >0 if stored value is greater than new value. If
+  /// flags.nullHandlingMode is StopAtNull, returns std::nullopt if either
+  /// 'left' or 'right' value is null or contains a null. If
```

Comment:
> 'left' or 'right'

please update; there are no left/right in this context

Reply:
Fixed